RecoParticleFlow/PFProducer/src/PFEGammaFilters.cc

```diff
@@ -164,6 +165,16 @@ bool PFEGammaFilters::passElectronSelection(const reco::GsfElectron& electron,
     }
   }
 
+  //TEMPORARY hack.
```

Comment:
please clarify in the code comment: temporary until when, to be fixed by who? better yet, link a github issue in the comment.